apigee_m10n.module

```diff
@@ -75,3 +101,13 @@ function apigee_m10n_form_user_admin_permissions_alter(&$form, FormStateInterfac
 function apigee_m10n_user_role_presave(EntityInterface $entity) {
   \Drupal::service('apigee_m10n.monetization')->userRolePresave($entity);
 }
+
+/**
+ * Implements hook_preprocess_field().
+ */
+function apigee_m10n_preprocess_field(&$variables) {
+  // Add a library to the rate plan `futurePlanLinks` field.
+  if ($variables['field_name'] === 'futurePlanLinks') {
+    $variables['#attached']['library'][] = 'apigee_m10n/rate_plan.future_links_field';
```

Comment:
`futurePlanLinks` is a link field. It isn't using a custom formatter so we don't have access tot he render array. Do you think it's worth creating a custom formatter in order to be able to avoid the pre-process?

Reply:
No, I would put the CSS all in one library. In fact, I would put it the same file as the entity styles, because it’s part of the entity view, even if one of those fields might not be there. That makes only one library to override when these styles are not wanted instead of 3. If someone is really bothered by the tiny amount of CSS that’s in the aggregated version of this, they can override it, but I think it actually is less harm to keep it all in one in this case.  The reason being that some rare plan pages will have it and some won’t, so the aggregated bundle will be different depending on this field, which probably has more of an impact. 
 
But, TBH, I wouldn’t include any of this CSS in the module for these fields at all.  I would use a table for the tabular data (volume bands), and rely on field classes for the rest.

Reply:
I think it makes sense to keep these separate and let aggregation handle it.